### PR TITLE
Update wizard.php

### PR DESCRIPTION
**Pull request by BenniGotchi**
Fix fatal error in Install wizard, when trying to import Lansuite XML dump

Inconsitency in variable name removed

refs #440

### CODE REVIEW

**Comment by BenniGotchi** [comment]
@M4LuZ, @andygrunwald : Any Idea, why the automatic checks fail?
There are some warnings about line lenght:

    23 | WARNING | Line exceeds 120 characters; contains 137
     |         | characters (Generic.Files.LineLength.TooLong)

Is this causing the fail?

**Comment by andygrunwald** [comment]
@BenniGotchi The check fails because of this recent change: https://github.com/lansuite/lansuite/pull/403#issuecomment-473046255

File: modules/install/wizard.php [hunk 1]
```diff
@@ -4,8 +4,8 @@
     $db->success = 0;
 }
 $importXml = new \LanSuite\XML();
-$installImport = new \LanSuite\Module\Install\Import($importXml);
-$install = new \LanSuite\Module\Install\Install($installImport);
```
Comment (andygrunwald) [inline modules/install/wizard.php]: Maybe I don't see it, but there is only the change of the variable name, right?
What does the Fatal error say? Or why will another name fix it?

Comment (andygrunwald) [reply modules/install/wizard.php]: Ok. Saw it here. Sorry: https://github.com/lansuite/lansuite/issues/440

**Pull request merged**